packages/esm-active-visits-app/src/types/index.ts

```diff
@@ -26,3 +28,28 @@ export interface Identifier {
   preferred: boolean;
   voided: boolean;
 }
+
+export interface Visit {
+  uuid: string;
+  display?: string;
+  encounters: Array<OpenmrsResource>;
+  patient?: OpenmrsResource;
+  visitType: VisitType;
+  location?: Location;
+  startDatetime: string;
+  stopDatetime?: string;
+  attributes?: Array<OpenmrsResource>;
+  [anythingElse: string]: any;
+}
+
+export interface Location {
+  uuid: string;
+  display?: string;
+  name?: string;
+}
+
+export interface VisitType {
+  uuid: string;
+  display: string;
+  name?: string;
+}
```

Comment:
If these 2 types are from the api then your better off deleting them and using the `OpenmrsResource` which comes with `display` and `uuid`